website_legal_page/migrations/13.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 Alexandre Díaz - Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from copy import deepcopy
+
+from lxml import etree as ElementTree
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+OLD_VIEW_XML_IDS = [
+    "website_legal_page.advice",
+    "website_legal_page.privacy-policy",
+    "website_legal_page.terms-of-use",
+    "website_sale.terms",
+]
+
+
+def _merge_old_views(env):
+    # Gets the old views that was changed by user (has a website_id)
+    old_view_ids = env["ir.ui.view"].search(
+        [
+            ("key", "in", OLD_VIEW_XML_IDS),
+            ("website_id", "!=", False),
```

Comment:
Good question... but ok, if you think that old samples need be merged, i do it. No problem.
In case of merge old samples (no modified by user) i will preserve the alert message "THIS IS A SAMPLE".